include/openssl/e_os2.h

```diff
@@ -277,7 +277,8 @@ typedef unsigned __int64 uint64_t;
 # endif
 
 /* Format specifier for printing size_t */
-# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L)
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
+      && !defined(__VMS)
```

Comment:
With 4f3015bb30b7d95bb97408776b70e6a35fb91e8a, the VMS check isn't necessary any more.  I've checked.